src/Symfony/Component/Workflow/Workflow.php

```diff
@@ -235,6 +235,23 @@ public function getEnabledTransitions(object $subject)
         return $enabledTransitions;
     }
 
+    /**
+     * {@inheritdoc}
```

Comment:
It is not inside the interface thus no inherited doc here i fear

Reply:
you're right !
thank you :)